### PR TITLE
added new fields

### PR DESCRIPTION
**Pull request by TrevorOctober**
@KaitCrawford Please review

### CODE REVIEW

File: dump.rdb [hunk 1]
```diff
@@ -0,0 +1 @@
+REDIS0006�ܳC�Z��V
```
Comment (KaitCrawford) [inline dump.rdb]: Please don't commit this file. Delete it or add it to .gitignore or something

File: mentorship/mentor/models.py [hunk 2]
```diff
@@ -24,6 +24,12 @@ class Mentee(Contact):
 class Mentor(Contact):
     mentee = models.OneToOneField(Mentee, related_name="mentee", null=True,
                                   blank=True)
+    profile_pic = models.ImageField(upload_to="media/", null=True, blank=True)
+    job_sector = models.CharField(max_length=30, null=True, blank=True)
+    job_title = models.CharField(max_length=30, null=True, blank=True)
+    motivation = models.CharField(max_length=200, null=True, blank=True)
+    inspiration = models.CharField(max_length=200, null=True, blank=True)
+    tags = models.CharField(max_length=200, null=True, blank=True)
```
Comment (KaitCrawford) [inline mentorship/mentor/models.py]: My gut feeling is that you should probably increase the max_length on these. Make the shorter ones about 100 and the longer ones you can either change to TextField or make the max_length something like 500

**Comment by TrevorOctober** [comment]
@KaitCrawford Ready for re-review

**Comment by KaitCrawford** [comment]
:+1: Looks good

**Pull request merged**